lib/info.gd

```diff
@@ -131,6 +131,7 @@ DeclareOperation("InfoLevel", [IsInfoClass]);
 DeclareGlobalFunction("SetInfoHandler");
 DeclareGlobalFunction("DefaultInfoHandler");
 
+
```

Comment:
Why is this empty line inserted in a file that otherwise is not modified at all in this PR?